### ISSUE TITLE
bug(YouTube): TalkBack accessibility label improvements

### ISSUE THREAD

**Issue by bzkmsjy** [MorpheApp/morphe-patches#3194]
### Bug description

Hello, I would like to continue submitting some accessibility label-related issues. All of the following issues were tested on TalkBack 17.0, using a Samsung S25 with One UI 8.5, a Xiaomi 10T with PixelOS 15, and a Redmi K50 with HyperOS 2.0.

1. Regarding the issue previously reported in #3096: the floating button added by the Voice over translation feature has been partially fixed, as it now has an accessibility label. However, I believe further improvements are needed. TalkBack currently does not announce whether this button is on or off. This makes it difficult for visually impaired users to determine whether the feature is currently enabled. Therefore, I hope the button can be given an accessibility state that allows TalkBack to announce whether it is on or off.

2. When entering the YouTube or YouTube Music settings and then opening the Morphe submenu, the back button in the upper-left corner has the correct accessibility label, “Navigate up,” on Samsung devices running One UI 8.5. However, on non-Samsung devices, this back button does not have an accessibility label.

3. In the same Morphe submenu, there is a search button in the upper-right corner. This button does not have an accessibility label on any of the Android systems I tested.

4. After tapping the search button mentioned in point 3 and searching for any option, a search history entry is created. To the right of each search history entry, there is a trash-can-shaped delete button. This button does not have an accessibility label.

Thank you for your continued efforts to improve accessibility. I hope these issues can be addressed in future updates.

### App version patched

YouTube 21.16.256

### Error logs

```shell
No need
```

### Acknowledgements

- [x] I have checked all open and closed bug reports and this is not a duplicate.
- [x] All requested information has been provided properly.
